mqtt-streaming/src/main/scala/akka/stream/alpakka/mqtt/streaming/impl/ClientState.scala

```diff
@@ -198,6 +198,8 @@ import scala.util.{Failure, Success}
           data.settings
         )
       )
+    case (_, ConnectionLost) =>
+      Behavior.same
```

Comment:
I was accidentally stashing a lost connection while being in a disconnected state. No need - and indeed it can then lead to a valid connection being cancelled.